cinema/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class CinemaConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
```

Comment:
your project should be one-styled, but you use double and single quotes in this project. Please run blank for the whole project.

Reply:
fixed.